operational_analysis/types/asset.py

```diff
@@ -1,17 +1,23 @@
 import importlib
 import itertools
 
-import geopandas as gp
 import numpy as np
 import pandas as pd
+from pyproj import Transformer
 from shapely.geometry import Point
 
 
 class AssetData(object):
     """
     This class wraps around a GeoPandas dataframe that contains
     metadata about the plant assets. It provides some useful functions
-    to work with this data (e.g., calculating nearest neighbors, etc.)
+    to work with this data (e.g., calculating nearest neighbors, etc.). 
+    
+    Note:
+        This class requires the geopandas package, which can be installed using:
+            pip install geopandas
+
+        If using Microsoft Windows, please see the 'Requirements' section of the OpenOA readme file for instructions on installing geopandas.
```

Comment:
@RHammond2 great that you found a good alternative to geopandas. I added this note because of the earlier temporary fix where we were still using geopandas but not making it a requirement. So I would remove these lines since they aren't relevant anymore.

Reply:
Good catch! I'll update the doctsring.